lib/response.js

```diff
@@ -229,7 +241,28 @@ res.send = function send(body) {
     this.end();
   } else {
     // respond
-    this.end(chunk, encoding);
+    if (isChunkBlob) {
+      var response = this;
+      try {
+        var WritableStream = require('stream/web').WritableStream;
+        chunk.stream().pipeTo(new WritableStream({
+          write: function(chunk) {
+            response.write(chunk,encoding);
+          },
+          close: function() {
+            response.end();
+          }
+        }))
+      } catch(error) {
+        chunk.arrayBuffer().then(function (result) {
+          response.end(Buffer.from(result), encoding);
+        }).catch(function (err) {
+          response.end();
```

Comment:
If there is an error, it should ideally be forwarded to the error handling pipeline rather than swallow and just ending the response wirh no info.

Reply:
done 👍

Reply:
It should forward the error to the error handling pipeline, not just output the error directly. May sites what users to see a customized error page and have the error written to their logs, not displayed to the end user.

Reply:
I just removed that catch callback so that if in case an error is thrown itself by the express then the user can handle it using try catch without ending it with no response.

Reply:
@dougwilson sorry to misunderstand you please check the current code where I pass the error through `req.next(err)`. I don't know whether I am right or wrong. if I am wrong please let me know the right way to do this.

Thank you 😊